examples/detr_wheat.py

```diff
@@ -30,16 +30,16 @@ def height(self, o) -> int:
     def width(self, o) -> int:
         return o.width
 
-    def label(self, o) -> List[int]:
+    def labels(self, o) -> List[int]:
         return [1]
 
-    def bbox(self, o) -> List[BBox]:
+    def bboxes(self, o) -> List[BBox]:
         return [self.bbox]
 
-    def area(self, o) -> List[float]:
-        return [self.bbox.area]
+    def areas(self, o) -> List[float]:
+        return [self.bbox.areas]
 
-    def iscrowd(self, o) -> List[bool]:
+    def iscrowds(self, o) -> List[bool]:
```

Comment:
The plural of crowd is still crowd: Crowd is uncountable noun. All "crowds" should be reverted to crowd

Reply:
Nice catch up!! Thanks!

This is a scary refactor, but I think it's necessary.

Reply:
Before I start the renaming, is it a problem that `iscrowd` is going to be associated both with a single item and a list of items?

Even if `iscrowds` is not a correct spelling, maybe it's a good idea to leave like so for consistency?

Reply:
It should be isCrowd. Crowd implicitly refers to many (equivalent to a list)

Reply:
It seems `crowds` is not technically wrong [source1](https://ell.stackexchange.com/questions/24616/difference-between-a-crowd-and-crowds) [source2](https://ell.stackexchange.com/questions/153754/should-i-fill-with-crowd-or-crowds-in-this-context) [source3](https://simple.wiktionary.org/wiki/crowds).

Let's stick with that because it makes a lot of intuitive sense, e.g. you might want to do `for iscrowd in record['iscrowds']`